librz/io/p/io_shm.c

```diff
@@ -106,10 +108,10 @@ static RzIODesc *shm__open(RzIO *io, const char *pathname, int rw, int mode) {
 		return NULL;
 	}
 	const char *ptr = pathname + 6;
-	shm->name = strdup(ptr);
+	shm->name = rz_str_newf("/%s", ptr);
```

Comment:
why this?

Reply:
for some oses it s mandatory the name starts with a slash to work correctly.

Reply:
isn't that accomplished by just doing `shm:///dev/shm/hello`?

Reply:
@ret2libc I'ts unportable, I think. Anyway, this method works everywhere except MacOS and Windows (https://github.com/rizinorg/rizin/issues/1758)
I have zero idea why OpenBSD CI fails though, maybe not enough memory to do the `mmap()`?

Reply:
that might be it however when I get OOM issue on OpenBSD with software X I usually get it on NetBSD too.

Reply:
How does the `shm://<xxxxx>` string look like for all major platforms?

Like, for Linux it is something like `shm:///dev/shm/hello`... For the others?

Reply:
It doesn't look like `shm:///dev/shm/hello` - it's `shm://psm_7effab` on Linux, for example

Reply:
Ok, the others?